### ISSUE TITLE
[Suggestion] A corner in the launcher for an preset image showcase

### ISSUE THREAD

**Issue by ibin7777** [Youda008/DoomRunner#86]
An optional frame in the doomrunner for showcasing image for the preset, 

Ideally can select custom image, and when not indicated - auto queue screenshots taken from that Preset.

That would be a nice touch for the bland-looking current state of the runner, and also helpful for identifying the game.

**Comment by ibin7777** [comment]
image example:
![example](https://user-images.githubusercontent.com/64576479/202108644-222b730b-57d4-4caf-a4b2-8d96e59df7e7.png)

**Comment by ibin7777** [comment]
Just found out duplicated issue 47

**Comment by Youda008** [comment]
https://github.com/Youda008/DoomRunner/issues/47? I think you mean https://github.com/Youda008/DoomRunner/issues/68, right?
I don't think that's a duplicate. If i understand it right, he's asking for an area that would show previews of all screenshots in a screenshot folder. Which is in my opinion pointless, because you can just use Windows file explorer for that.

Your suggestion makes sense, but i would need to dive into yet another unexplored area of Qt, and i'm affraid it would be a lot of work.
I'm not saying "no", but "not now", maybe some day.

**Comment by ibin7777** [comment]
> #47? I think you mean #68, right? I don't think that's a duplicate. If i understand it right, he's asking for an area that would show previews of all screenshots in a screenshot folder. Which is in my opinion pointless, because you can just use Windows file explorer for that.
> 
> Your suggestion makes sense, but i would need to dive into yet another unexplored area of Qt, and i'm affraid it would be a lot of work. I'm not saying "no", but "not now", maybe some day.

Yes 68 is what I meant, sorry for that, too many tabs opened is apparently not a good habit.
